pkg/operator/ceph/csi/csi.go

```diff
@@ -175,8 +175,14 @@ func (r *ReconcileCSI) setParams(ver *version.Info) error {
 		CSIParam.EnableNFSSnapshotter = false
 	}
 
-	CSIParam.EnableCSIAddonsSideCar = false
-	if strings.EqualFold(k8sutil.GetValue(r.opConfig.Parameters, "CSI_ENABLE_CSIADDONS", "false"), "true") {
```

Comment:
`CSIParam.EnableCSIAddonsSideCar = false` let's keep this one as it is the default value for the Reconcile

Reply:
done